src/managing-domains/managing-domain-ownership.md

```diff
@@ -0,0 +1,73 @@
+Management of second-level domains is performed via the [Registry smart contract](https://github.com/unstoppabledomains/dot-crypto/blob/master/contracts/Registry.sol). The smart contract is built upon [OpenZeppelin implementation](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721#ERC721Burnable) of the [ERC-721](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md) token standard.
+
+Entities that can control domains are defined by the ERC-721 standard:
+
+* **Owner.** This is a direct owner of a domain, which has full control in managing domain ownership and records.
+* **Operator.** Registry allows any user to set their operators, that can control all domains owned by a user. There can be multiple operators per user.
+* **Approved address.** A domain owner can set an approved address that can control one particular domain. Only one approved address per domain is allowed.
+
+There are five basic operations that affect domain ownership:
+
+* **Minting.** When a domain is first created, an initial domain owner is assigned. Minting domains is its own separate topic and won't be covered in this article.
+* **Transferring.** There are two possible ways of transferring a domain: the one that keeps resolution settings, and the one that resets them.
```

Comment:
```suggestion
* **Transferring.** There are two possible ways of transferring a domain: the one that keeps domain records, and the one that resets them.
```

I think this is easier to understand

Reply:
I didn't quite want to use "domain records" here because technically domain records are always kept